src/styles/utils/_variables.scss

```
@@ -0,0 +1,3 @@
+$nav-link-hover: #00acdc;
+$card-title-hover: #34568b;
+$card-increase-hover: scale(120%);
```

Do not overuse variables. If value is used once then there is no need to make variable out of it. Also name variables so they are more descriptive for example:
```suggestion
$color-blue-light: #00acdc;
$color-blue: #34568b;
```